react/week1/todo-list-app/src/components/Todo.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './Todo.css'
+function Todo(props) {
+    console.log(props);
```

Comment:
Just to keep a good habit, remember to remove your development logs from your final commit.